Datadog/Example/Debugging/Helpers/SwiftUI.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+extension Color {
+    /// Datadog purple.
+    static var datadogPurple: Color {
+        return Color(UIColor(red: 99/256, green: 44/256, blue: 166/256, alpha: 1))
+    }
+
+    static var rumViewColor: Color {
+        return Color(UIColor(red: 0/256, green: 107/256, blue: 194/256, alpha: 1))
+    }
+
+    static var rumResourceColor: Color {
+        return Color(UIColor(red: 113/256, green: 184/256, blue: 231/256, alpha: 1))
+    }
+
+    static var rumActionColor: Color {
+        return Color(UIColor(red: 150/256, green: 95/256, blue: 204/256, alpha: 1))
+    }
+
+    static var rumErrorColor: Color {
+        return Color(UIColor(red: 235/256, green: 54/256, blue: 7/256, alpha: 1))
+    }
+}
+
+@available(iOS 13.0, *)
+internal struct DatadogButtonStyle: ButtonStyle {
+    func makeBody(configuration: DatadogButtonStyle.Configuration) -> some View {
+        return configuration.label
```

Comment:
@maxep I'm wondering if this kind of styling is sth we could leverage in `SwiftUI` instrumentation? The way it works is by inserting:
```swift
.buttonStyle(DatadogButtonStyle())
```
in view declaration **globally**, which then runs this piece of code for all `SwiftUI` buttons. There are also distinct styles for lists and other elements.

Reply:
Interesting! This could be something usable for populating the name, But from what I see, the `PrimitiveButtonStyle` and `PrimitiveButtonStyleConfiguration` do not expose much. The only thing we could leverage is the `ButtonRole`, which can be cancel or destructive. Worth exploring tho 👍